http/server.lua

```diff
@@ -859,6 +859,10 @@ local function process_client(self, s, peer)
             hdrs['content-length'] = #body
         end
 
+        if hdrs['content-type'] == nil then
+            hdrs['content-type'] = 'text/plain; charset=utf-8'
```

Comment:
Nit: It is 'text/plain; utf-8' in the commit message, looks as the typo.

Reply:
I would also leave several words why default content type may be important from security point of view.

Reply:
Fixed.